packages/jest-jasmine2/src/setup-jest-globals.js

```diff
@@ -34,21 +34,54 @@ const addSuppressedErrors = result => {
   }
 };
 
+const addPassedExpectations = result => {
+  const {passedExpectations} = getState();
+  setState({passedExpectations: []});
+  result.passedExpectations = result.passedExpectations.concat(passedExpectations);
```

Comment:
Not that I know of, was just attempting to match behavior of Jasmine. I'll make it a number.